### ISSUE TITLE
Bazel with --incompatible_no_transitive_loads

### ISSUE THREAD

**Issue by laurentlb** [bazelbuild/rules_webtesting#325]
Build is failing when I run Bazel with `--incompatible_no_transitive_loads`. We plan to enable this flag by default soon. Could you please update your code?

I've seen this error message, which should go away if you update Gazelle. There may be other errors though.

```
ERROR: .../rules_webtesting/web/go_repositories.bzl:17:1: file '@bazel_gazelle//:deps.bzl' does not contain symbol 'go_repository'
```

**Comment by laurentlb** [comment]
cc @DrMarcII 
Please update the rules.

[`bazelisk --migrate`](https://github.com/philwo/bazelisk) can help you test other flags and detect other problems.
